src/joint.dia.cell.js

```diff
@@ -927,7 +935,7 @@ joint.dia.CellView = joint.mvc.View.extend({
         if (this._collection) {
             // we don't want to trigger event on model as model doesn't
             // need to be member of collection anymore (remove)
-            this._collection.trigger('batch:stop', { batchName: 'pointer' });
+            this._collection.stopBatch('pointer');
```

Comment:
There is no `stopBatch` method defined on `collection` now. It's a legacy code anyway and this should be changed in the following way.
```javascript
if (this.model.graph) {
  this.model.startBatch('pointer');
  this._graph = this.model.graph;
}
  		  
if (this._graph) {
  // we don't want to trigger event on model as model doesn't
  // need to be member of graph anymore (remove)
  this._graph.startBatch('pointer');
  this._graph = null;
}
```